docs/generated/packages/storybook/documents/storybook-interaction-tests.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Setting up Storybook interaction tests with Nx
+description: This guide explains how you can set up Storybook interaction tests on your Nx workspace.
+---
+
+# Setting up Storybook interaction tests with Nx
```

Comment:
We should use the title form with upper case words, forgot the proper name, but basically

"Setting up Storybook Interaction Tests with Nx"